docs/en/latest/plugins/clickhouse-logger.md

```diff
@@ -0,0 +1,158 @@
+| database         | string  | required   |               |         | The DB name to store log.                   |
+| logtable         | string  | required   |               |         | The table name.                             |
+| user             | string  | required   |               |         |clickhouse user.                             |
+| password         | string  | required   |               |         |clickhouse password.                         |
+| timeout          | integer | optional   | 3             | [1,...] | Time to keep the connection alive after sending a request.                   |
+| name             | string  | optional   | "clickhouse logger" |         | A unique identifier to identity the logger.                             |
+| batch_max_size   | integer | optional   | 100           | [1,...] | Set the maximum number of logs sent in each batch. When the number of logs reaches the set maximum, all logs will be automatically pushed to the clickhouse.  |
+| max_retry_count  | integer | optional   | 0             | [0,...] | Maximum number of retries before removing from the processing pipe line.        |
+| retry_delay      | integer | optional   | 1             | [0,...] | Number of seconds the process execution should be delayed if the execution fails.             |
+| ssl_verify       | boolean | optional   | true          | [true,false] | verify ssl.             |
+
+## How To Enable
+
+The following is an example of how to enable the `clickhouse-logger` for a specific route.
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+      "plugins": {
+            "clickhouse-logger": {
+                "user": "default",
+                "password": "a",
+                "database": "default",
+                "logtable": "test",
+                "endpoint_addr": "http://127.0.0.1:8123"
+            }
+       },
+      "upstream": {
+           "type": "roundrobin",
+           "nodes": {
+               "127.0.0.1:1980": 1
+           }
+      },
+      "uri": "/hello"
+}'
+```
+
+## Test Plugin
+
+> success:
+
+```shell
+$ curl -i http://127.0.0.1:9080/hello
+HTTP/1.1 200 OK
+...
+hello, world
+```
+
+## Metadata
+
+| Name             | Type    | Requirement | Default       | Valid   | Description                                                                              |
+| ---------------- | ------- | ----------- | ------------- | ------- | ---------------------------------------------------------------------------------------- |
+| log_format       | object  | optional    | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | Log format declared as key value pair in JSON format. Only string is supported in the `value` part. If the value starts with `$`, it means to get `APISIX` variables or [Nginx variable](http://nginx.org/en/docs/varindex.html). |
```

Comment:
Let's update the doc according to the latest http-logger's: https://github.com/apache/apisix/blob/82d17abf9987528856cbd20ce5f17f2ae0a799ce/docs/en/latest/plugins/http-logger.md?plain=1#L90

Reply:
will add.